flutter/lib/state/task_runner.dart

```diff
@@ -0,0 +1,348 @@
+          isTestMode: store.testMode,
+          resourceManager: resourceManager,
+          backendBridge: backendBridge,
+          benchmark: benchmark,
+          runMode: accuracyMode,
+          commonSettings: backendInfo.settings.commonSetting,
+          backendLibName: backendInfo.libName,
+          logParentDir: currentLogDir,
+        ).run();
+
+        final accuracyResult = accuracyRunInfo.result;
+        benchmark.accuracyModeResult = BenchmarkResult(
+          // loadgen doesn't calculate latency for accuracy mode benchmarks
+          // so throughput is infinity which is not a valid JSON numeric value
+          throughput: 0.0,
+          accuracy: accuracyResult.accuracy1,
+          accuracy2: accuracyResult.accuracy2,
+          backendName: accuracyResult.backendName,
+          acceleratorName: accuracyResult.acceleratorName,
+          batchSize: benchmark.benchmarkSettings.batchSize,
+          validity: false,
+        );
+      }
+
+      final resultHelper = ResultHelper(
+        benchmark: benchmark,
+        backendInfo: backendInfo,
+      );
+
+      exportResults.add(resultHelper.exportResultFromRunInfo(
+        performanceInfo: performanceRunInfo,
+        accuracyInfo: accuracyRunInfo,
+        perfMode: perfMode,
+        accuracyMode: accuracyMode,
+      ));
+    }
+
+    if (aborting) {
+      aborting = false;
+      return null;
+    }
+    return ExtendedResult(
+      meta: ResultMetaInfo(uuid: const Uuid().v4()),
+      environmentInfo: DeviceInfo.instance.envInfo,
+      results: exportResults,
+      buildInfo: BuildInfoHelper.info,
+    );
+  }
+}
+
+class _NativeRunHelper {
+  final bool enableArtificialLoad;
+  final BridgeIsolate backendBridge;
+
+  final Benchmark benchmark;
+  final BenchmarkRunMode runMode;
+  final String logDir;
+  late final RunSettings runSettings;
+
+  _NativeRunHelper({
+    required this.enableArtificialLoad,
+    required this.backendBridge,
+    required this.benchmark,
+    required this.runMode,
+    required bool isTestMode,
+    required ResourceManager resourceManager,
+    required List<pb.Setting> commonSettings,
+    required String backendLibName,
+    required String logParentDir,
+  }) : logDir = '$logParentDir/${benchmark.id}-${runMode.readable}' {
+    runSettings = benchmark.createRunSettings(
+      runMode: runMode,
+      resourceManager: resourceManager,
+      commonSettings: commonSettings,
+      backendLibName: backendLibName,
+      logDir: logDir,
+      isTestMode: isTestMode,
+    );
+  }
+
+  Future<RunInfo> run() async {
+    await Directory(logDir).create(recursive: true);
+
+    Cancelable<void>? artificialLoadHandle;
+    if (enableArtificialLoad) {
+      print('Apply the artificial CPU load for ${benchmark.taskConfig.id}');
+      const value = 999999999999999.0;
+      artificialLoadHandle =
+          Executor().execute(arg1: value, fun1: _doSomethingCPUIntensive);
```

Comment:
Got this error with the artificial load enabled:
```
flutter: image_classification: performance mode: starting...
[VERBOSE-2:dart_vm_initializer.cc(41)] Unhandled Exception: Invalid argument(s): Illegal argument in isolate message: (object is a SuspendState)
#0      _SendPortImpl._sendInternal (dart:isolate-patch/isolate_patch.dart:249:43)
#1      _SendPortImpl.send (dart:isolate-patch/isolate_patch.dart:230:5)
#2      WorkerImpl.work (package:worker_manager/src/worker/worker_io.dart:60:15)
#3      _Executor._schedule (package:worker_manager/src/scheduling/executor.dart:162:26)
#4      _Executor.execute.executing (package:worker_manager/src/scheduling/executor.dart:125:9)
#5      _Executor.execute.<anonymous closure> (package:worker_manager/src/scheduling/executor.dart:142:71)
#6      Cancelable.thenNext.<anonymous closure> (package:worker_manager/src/cancelable/cancelable.dart:77:27)
<asynchronous suspension>
#7      Cancelable.thenNext.<anonymous closure> (package:worker_manager/src/cancelable/cancelable.dart:73:28)
<asynchronous suspension>
```

This one should be fixed in this PR.

Reply:
I changed artificial load stopping back to `Executor().dispose()`, it should be fixed now.

Reply:
Please test it to make sure it's really fixed.
BTW, the issue is caused by the method `_doSomethingCPUIntensive()`, which is changed to a class instance method. 
> It needs to be either a static function or a top level function to be accessible as a Flutter entry point.

Reply:
You can also add this as a comment to the function `_doSomethingCPUIntensive()`:

> This function needs to be either a static function or a top level function to be accessible as a Flutter entry point.

Reply:
> the issue is caused by the method `_doSomethingCPUIntensive()`

Fixed.

It also seems like `artificialLoadHandle.cancel` doesn't work, so I still kept the `dispose` call.
It seems to work to me now, CPU load changes how I expect it to.